admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,256 @@
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+----
+<1> The number of estimated pods that can be scheduled on the cluster. 
+
+[[admin-guide-running-cluster-capacity-inside-pod]]
+== Running Cluster Capacity as a Job Inside of a Pod
+
+Running the cluster capacity tool as a job inside of a pod has the advantage of
+being able to be run multiple times without needing user intervention, or to be
+compatible with xref:../dev_guide/cron_jobs.adoc#dev-guide-cron-jobs[cron jobs].
+Running the cluster capacity tool as a job involves using a `ConfigMap`.
+
+. Create the cluster role:
++
+----
+$ cat << EOF| oc create -f -
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cluster-capacity-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "nodes", "persistentvolumeclaims", "persistentvolumes", "services"]
+  verbs: ["get", "watch", "list"]
+EOF
+----
+
+. Create the service account:
++
+----
+$ oc create sa cluster-capacity-sa
+----
+
+. Add the role to the service account:
++
+----
+$ oadm policy add-cluster-role-to-user cluster-capacity-role \
+    system:serviceaccount:default:cluster-capacity-sa
+----
+
+. Define and create the pod specification:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+----
+
+. Create the job using the below example of a job specification file:
++
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cluster-capacity-job
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: cluster-capacity-pod
+    spec:
+        containers:
+        - name: cluster-capacity
+          image: openshift/origin-cluster-capacity
+          imagePullPolicy: "Always"
+          volumeMounts:
+          - mountPath: /test-pod
+            name: test-volume
+          env:
+          - name: CC_INCLUSTER <1>
+            value: "true"
+          command:
+          - "/bin/sh"
+          - "-ec"
+          - |
+            /bin/cluster-capacity --podspec=/test-pod/pod.yaml --verbose
+        restartPolicy: "Never"
+        serviceAccountName: cluster-capacity-sa
+        volumes:
+        - name: test-volume
+          configMap:
+            name: cluster-capacity-configmap
+----
+<1> A required environment variable letting the cluster capacity tool
+ know that it is running inside a cluster as a pod.
++
+[NOTE]
+====
+The cluster capacity analysis is mounted in a volume using a `ConfigMap` named
+`cluster-capacity-configmap` to mount input pod spec file `pod.yaml` into a
+volume `test-volume` at the path `/test-pod`.
+
+To create the `ConfigMap`:
```

Comment:
I think move it before Job discussion and after input pod spec as configmap is used to store input config map.